runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/InfoBox.java

```diff
@@ -81,4 +83,14 @@ public boolean cull()
 	{
 		return false;
 	}
+
+	public boolean getOutline()
```

Comment:
Rather than writing get/set methods by hand use `@Getter` and `@Setter` annoations on the `outline` field

Reply:
Thanks for the tip :+1: 

But for now I've removed the possibility to change this on the InfoBox class directly, and made it so InfoBoxOverlay decides if it should be outlined and/or have a background.